include/picongpu/param/pml.param

```diff
@@ -0,0 +1,158 @@
+/* Copyright 2019 Sergei Bastrakov
+ *
+ * This file is part of PIConGPU.
+ *
+ * PIConGPU is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * PIConGPU is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PIConGPU.
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *
+ * Configure the perfectly matched layer (PML).
+ *
+ * To enable PML use YeePML field solver.
+ */
+
+#pragma once
+
+
+namespace picongpu
+{
+namespace fields
+{
+namespace maxwellSolver
+{
+namespace yeePML
+{
+
+    /* The parameters in this file are only used if field solver is YeePML.
```

Comment:
Okay - perhaps: `if the field solver is set to YeePML` or  `if the field solver selected is YeePML`.

Reply:
Good point.